### ISSUE TITLE
The enum generation only works with direct classes of Enum. 

### ISSUE THREAD

**Issue by gmazzei-chartboost** [christianhelle/autofaker#11]
The current code to generate an enum is the following:
```
def is_enum(t) -> bool:
    return t.__base__.__name__ == 'Enum'
```
This allows only direct classes of Enum to be generated. If you have something like 
```
class BasicEnum(Enum):
   some stuff

class MyEnum(BasicEnum):
   VALUE_1 = 1
   VALUE_2 = 2
```

it wont work.
This can be fixed by just asking if it is instance of Enum instead

**Comment by christianhelle** [comment]
@gmazzei-chartboost thanks for taking the time to report this. I'm unfortunately on my summer vacation and won't be able to look into this in detail in the next 2 weeks.

If you know the fix and don't mind creating a PR yourself then I will approve and merge it from my phone and make sure it gets released to PyPi on the same day

**Comment by christianhelle** [comment]
The fix for this is released and published as [v1.0.15](https://pypi.org/project/autofaker/1.0.15/) to PyPi
